src/Symfony/Component/Console/Tests/Command/CommandTest.php

```diff
@@ -293,6 +293,22 @@ public function testSetCode()
         $this->assertEquals('interact called'.PHP_EOL.'from the code...'.PHP_EOL, $tester->getDisplay());
     }
 
+    public function testSetCodeBindToClosure()
+    {
+        if (PHP_VERSION_ID < 50400) {
+            $this->markTestSkipped('Test skipped, for PHP 5.4+ only.');
+        }
+
+        $command = new \TestCommand();
+        $ret = $command->setCode(function (InputInterface $input, OutputInterface $output) {
+            $output->writeln(get_class($this));
+        });
+        $this->assertEquals($command, $ret, '->setCode() implements a fluent interface');
```

Comment:
This assertion is not really part of this test case. I'd at least move it to the end with the other assertion. The best would be to move it to a separate test case (which could be run on all PHP versions btw).